internals/cli/cmd_run.go

```diff
@@ -31,18 +31,6 @@ import (
 	"github.com/canonical/pebble/internals/systemd"
 )
 
-var shortRunHelp = "Run the pebble environment"
-var longRunHelp = `
-The run command starts pebble and runs the configured environment.
```

Comment:
Curious that most are being moved up while this one is being moved down.

Reply:
That's because I was trying to keep the help strings on top of the `cmdFoo` struct, not necessarily on the absolute top of the file. But I agree that it should be, thanks.